README.md

```diff
@@ -36,14 +36,15 @@ The main file is `sync_ratings.py`.
 Usage description:
 ```
 usage: sync_ratings.py [-h] [--dry] [--reverse] [--log LOG] [--passwd PASSWD] 
-                       [--player PLAYER] --server SERVER --username USERNAME
+                       [--sync ITEM] [--player PLAYER] --server SERVER --username USERNAME
 
 Synchronizes ID3 music ratings with a Plex media-server
 
 optional arguments:
   -h, --help           show this help message and exit
   --dry                Does not apply any changes
-  --reverse            Reverses ratings synchronization from Plex to local player
+  --reverse            Reverses ratings synchronization from Plex to local player 
+  --sync 			   Selects which items to sync: TRACKS, PLAYLISTS, or BOTH [default is TRACKS]
```

Comment:
use spaces to indent here, you used tabs

Reply:
also I think it would be better to make `sync` a list. Take a look at `argparse`'s `nargs` option: https://docs.python.org/3/library/argparse.html#index-2
Then you could call it like `--sync PLAYLISTS TRACKS` if you want both

Reply:
would you keep both as an option as well?  so you could do all of the following as acceptable options. 
--sync playlists
--sync tracks
--sync both
--sync tracks playlists

Reply:
there is a choice here - i can add a list which would allow --sync playlists tracks but would break the case sensitivity protection

Reply:
I would remove `both` as an option. Maybe you want to sync other things :man_shrugging: . What you could do is the following:
```python
sync_items = ["TraCKs", "playLists"]  # this is the output of argparse
for sync_item in sync_items:
    if sync_item.lower() == "tracks":
        sync_tracks()
    elif sync_item.lower() == "playlists":
        sync_playlists()
    else:
        raise ValueError(f"Unsupported sync target {sync_item}")
```

Reply:
OK, I see what you are saying now.  I was attempting to use argparse choices feature which was case sensitive.
I'll have to figure out how nargs works.  

thanks for the pointers.

Reply:
cool. Granted, I am not 100% sure if this will work with keyword (i.e. `--something`) arguments. In the worst case, you need to make it a positional (i.e. at the end of the command) _or_ you say this is a single argument (to argparse) and later split it manually like
```python
sync_items = self.options.sync.split(',')
```
This would do the following:
```python
>>> sync_argument = "tracks,playlists"
... sync_items = sync_argument.split(",")
... print(sync_items)
["tracks", "playlists"]
```